packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -52,6 +53,18 @@ Future<vm_service.VmService> _kDefaultFuchsiaIsolateDiscoveryConnector(Uri uri)
   return connectToVmService(uri);
 }
 
+Future<DartDevelopmentService> _kDefaultFuchsiaIsolateDiscoveryDDSConnector(
+  Uri remoteVmServiceUri,
+  Uri serviceUri) {
+  return DartDevelopmentService.startDartDevelopmentService(
```

Comment:
nit: 2 space indent like the reset of the tool

Reply:
I really wish you guys used `dartfmt` :(

Reply:
yup